.github/workflows/build-image-ui-pr.yml

```diff
@@ -0,0 +1,25 @@
+name: Test UI container image build and deployment
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - "clients/ui/**"
+      - "!LICENSE*"
+      - "!DOCKERFILE*"
+      - "!**.gitignore"
+      - "!**.md"
+      - "!**.txt"
+env:
+  IMG_ORG: kubeflow
+  IMG_REPO: model-registry
+  PUSH_IMAGE: false
+  BRANCH: ${{ github.base_ref }}
+jobs:
+  build-image:
+    runs-on: ubuntu-latest
+    steps:
+      # checkout branch
+      - uses: actions/checkout@v4
+      - name: Build UI Image
+        shell: bash
+        run: ./scripts/build_deploy.sh
```

Comment:
I'm a bit confused, the PR body specs:

> Runs an image build of the UI container on PR.

this invokes `./scripts/build_deploy.sh`

which invokes:

https://github.com/kubeflow/model-registry/blob/c27acc22d4ab77e38b0bf5495f09b1dba4930624/scripts/build_deploy.sh#L49-L57

which invokes:

https://github.com/kubeflow/model-registry/blob/c27acc22d4ab77e38b0bf5495f09b1dba4930624/Makefile#L263-L266

which is

https://github.com/kubeflow/model-registry/blob/main/Dockerfile

which is the Model Registry Go REST server.

What am I missing, please?

Reply:
bump @alexcreasy @lucferbux

Reply:
Oh wow, I'm sorry @tarilabs good catch, I swear at some point it had `IMG_REPO: model-registry` and I blanked out readiing that this time.
Yes, @alexcreasy this should be `model-registry-ui`, if not, you are building model registry, not the UI.
Great catch @tarilabs